resources/views/ResultFields/AutoCompleteListItem.fields.json

```diff
@@ -0,0 +1,20 @@
+[
+    "images.all.url",
```

Comment:
@stentrop benötigen wir hier wirklich die Bilder in voller Größe? Meiner Meinung nach sollte die Preview hier ausreichen.